src/backend/gporca/libnaucrates/src/init.cpp

```diff
@@ -122,23 +120,13 @@ void
 gpdxl_init()
 {
 	// create memory pool for Xerces global allocations
-	{
```

Comment:
Add in the description why the gpdxl_init() function has been modified. This changes do not fix errors at processing of OOM and exceptions

Reply:
The gpdxl_init function has been changed, because the auto memory pool is used to automatically free memory when it goes out of scope, but here it is immediately removed from it, and in fact, it is not needed at all.